packages/deploy/src/api/deployment-config.ts

```diff
@@ -0,0 +1,36 @@
+import { PlatformApiClient } from './platform';
+import { DeploymentConfigCreateRequest, DeploymentConfigResponse, RemoveResponse } from '../models';
+
+export class DeploymentConfigClient extends PlatformApiClient {
+  public async get(deploymentConfigId: string): Promise<DeploymentConfigResponse> {
+    return this.apiCall(async (api) => {
+      return await api.get(`/deployment-config/${deploymentConfigId}`);
+    });
+  }
+  public async list(): Promise<DeploymentConfigResponse[]> {
+    return this.apiCall(async (api) => {
+      return await api.get(`/deployment-config`);
```

Comment:
Nitpick: To minimise errors we can move `/deployment-config` to constant variable at top of the file and use it to compute urls where needed.